.github/workflows/test.yml

```diff
@@ -42,5 +42,17 @@ jobs:
         run: npm run prettier:check
       - name: Test
         run: npm test
+      # N.B.: The selenium server cannot run as a GitHub Actions "service" as
+      # the tests need to mount local data into the container.
+      - name: Start Selenium Server
```

Comment:
This adds roughly 30s to every run, which seems ok, but also not super fast. Do you know of a faster way to get this up and running?

Reply:
It seems GitHub Actions keeps some Docker images in a cache (which is nice), but they won't tell us which ones (or at least I can't find it): https://github.com/actions/virtual-environments/blob/main/docs/software-and-images-guidelines.md#software-and-images-support-policy

Reply:
We can always optimize for performance later. It's great to have this in place 👍